packages/test/local-server-stress-tests/src/ddsOperations.ts

```diff
@@ -24,6 +24,12 @@ export interface DDSModelOp {
 	op: unknown;
 }
 
+export interface OrderSequentially {
+	type: "orderSequentially";
+	operations: DDSModelOp[];
+	rollback: boolean;
```

Comment:
Would be good to at least document what this rollback flag means if not the whole interface.  Maybe
```suggestion
	/** Induce a rollback after playing all operations */
	rollback: boolean;
```